Vagrantfile

```diff
@@ -0,0 +1,131 @@
+# -*- mode: ruby -*-
```

Comment:
I have added comments about the QI files throughout "the stack".  See [GPII-1988](https://issues.gpii.net/browse/GPII-1988).